### DISCUSSION TITLE
footer, navbar and content bar

### DISCUSSION THREAD

**Discussion by Gordolin** [hbstack/.github#38] [Issues and bugs]
Hi, the left navbar and the right content bar run out of the browser, when you scroll down to the bottom of a page. It seems, that it is exactly the hight of the footer bar. Could you fix this please.

**Comment by razonyang** [comment]
Cloud you please provide detailed information like live URL, screenshot, and browser?

**Reply by Gordolin** [reply]
Hi, i hav no public site at this moment. But you can see this behavior on https://hbstack.dev/modules/docs/content-panel/
![2023-12-20 18_17_31-](https://github.com/hbstack/.github/assets/120491875/136565f7-511c-4cf1-91a9-d297da801730)

I tried this with chrome and brave browser with windows 11

**Comment by razonyang** [comment]
That's expected (how it was designed), the left navbar will be sticky until the page reaches the footer, so I'm closing it as non-bug.
